src/contracts/pods/EigenPodManager.sol

```diff
@@ -57,12 +58,14 @@ contract EigenPodManager is
     function initialize(
         uint256 _maxPods,
         IBeaconChainOracle _beaconChainOracle,
+        IFunctionGateway _functionGateway,
```

Comment:
do you want to maybe just make a setter function for this instead of including in the initializer?
I guess it depends what you find to be easier in tests, but if we won't be reinitializing anywhere, then either option works